dipy/align/imaffine.py

```diff
@@ -967,23 +979,47 @@ def _init_optimizer(self, static, moving, transform, params0,
                                                  self.sigmas,
                                                  moving_grid2world,
                                                  moving_spacing, False)
+            if moving_mask:
+                self.moving_mask_ss = IsotropicScaleSpace(
+                    moving_mask, self.factors,
+                    [0]*len(self.factors),
```

Comment:
Here I set the smoothing factors to zero, as it is a mask. But it might be better to smooth, interpolate, and threshold. What do you think?